odk1-src/central-backup.rst

```diff
@@ -3,6 +3,13 @@
 Backing Up Central
 ==================
 
+Having a data backup strategy is a critical part of running a web service like ODK Central. Backups should go to a system in a different physical location from where Central is installed in order to prevent data loss across a broad range of scenarios. Many cloud providers offer backup strategies that run automatically. If you are an experienced system administrator, you may want to set up your own backups of the PostgreSQL database. If you don't already have a server-wide backup system in place and don't want to set up your own database backup, Central provides a managed backup system to Google Drive.
```

Comment:
"of the PostgreSQL database that contains all Central's data."

Basically, we need a lightweight way of saying that everything that is in Central is in that database. That is, there is nothing else important on the filesystem.